docs/preset-env.md

```diff
@@ -302,7 +328,38 @@ var b = new Map();
 
 #### `useBuiltIns: false`
 
-Don't add polyfills automatically per file, or transform `import "@babel/polyfill"` to individual polyfills.
+Don't add polyfills automatically per file, or transform `import "core-js"` to individual polyfills.
+
+### `corejs`
+
+`CorejsVersion` or `{ version: CorejsVersion, proposals: boolean }`, defaults to `2`.
+`CorejsVersion` can be either `2` or `3`.
+
+This option, which can only be used when `useBuiltIns` is not `false`, configures `@babel/preset-env` to inject imports to the correct `core-js` version.
+By default, only polyfills for stable ECMAScript features are injected: if you want to polyfill them, you need to set `proposals: true`.
+
+`corejs: 2` only supports global variables (e.g. `Promise`) and static properties (e.g. `Array.from`), while `corejs: 3` also supports instance properties (e.g. `[].includes`).
+
+By default, `@babel/plugin-transform-runtime` doesn't polyfill proposals. If you are using `corejs: 3`, you can opt-in using the `proposals: true` boolean flag:
```

Comment:
Seems unnecessary/redundant, we mention this above?

> By default, only polyfills for stable ECMAScript features are injected: if you want to polyfill them, you need to set `proposals: true`.

Reply:
Yeah, I think I accidentally copied these last two sentences from the transform-runtime docs